docs/design/autoware-interfaces/components/perception-interface.md

```diff
@@ -0,0 +1,35 @@
+# Perception
+
+![Node diagram](images/Perception-Bus-ODD-Architecture.drawio.svg)
+
+## Inputs
+
+### PointCloud
+
+PointCloud data published by Lidar.
+
+- [sensor_msgs/PointCloud2](http://docs.ros.org/en/api/sensor_msgs/html/msg/PointCloud2.html)
+
+### Image
+
+Image frame captured by camera.
+
+- [sensor_msgs/Image](http://docs.ros.org/en/api/sensor_msgs/html/msg/Image.html)
+
+### Vehicle kinematic state
+
+current position of ego, used in traffic signals recognization. See Iutputs of Planning.
+
+### Lanelet2 Map
+
+map of the environment. See Iutputs of Planning.
+
+## Output
+
+### 3D Object Predictions
+
+3D Object detected, tracked and predicted by sensor fusing. See Iutputs of Planning.
```

Comment:
Since this is output of perception module, let's write the message type here and have Planning interface design doc refer to this page.

(Please do the same for Traffic Light Result)